io.github.masterbillwoo.rise_of_the_penguins_gb.yml

```diff
@@ -0,0 +1,32 @@
+app-id: io.github.masterbillwoo.rise_of_the_penguins_gb
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.4'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+
+modules:
+  - name: rise-of-the-penguins-gb
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 rise.of.the.penguins.gb.pck ${FLATPAK_DEST}/share/rise-of-the-penguins-gb/godot-runner.pck
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/masterbillwoo/rise-of-the-penguins-gb-linux-pck-export/releases/download/v1.0.0/rise.of.the.penguins.gb.pck
+        sha256: 20E83BF793CC424CE7931F6AB66CE5F7602AF759A79544E2495586405E929AB9
+      - type: file
+        path: ${FLATPAK_ID}.desktop
+      - type: file
+        path: ${FLATPAK_ID}.png
+      - type: file
+        path: ${FLATPAK_ID}.metainfo.xml
```

Comment:
this is even worse than before. `${FLATPAK_ID}.desktop` doesn't exist. Not the other 2.